sdk/go/dagger/error.go

```diff
@@ -0,0 +1,11 @@
+package dagger
+
+import "fmt"
+
+const (
+	ErrorHelpBlurb = "Please visit https://dagger.io/help#go for troubleshooting guidance."
```

Comment:
Updated with the URL from here: https://github.com/dagger/dagger.io/pull/980#pullrequestreview-1141566520

Including the `#go` as suggested here: https://github.com/dagger/dagger/issues/3316#issuecomment-1276890534

Let me know if you agree with the phrasing

Let me know if you agree with the phrasing here.